add Tilt support (#60)
deislabs/smi-adapter-istio: README.md

## Contributing

> Please refer to [CONTRIBUTING.md](./CONTRIBUTING.md) for more information on contributing to the specification and setting up your development environment.

Reviewer: ```suggestion
Please refer to [CONTRIBUTING.md](./CONTRIBUTING.md) for more information on contributing to this repository and setting up your development environment.
```
